dev/scss/_home.scss

```diff
@@ -11,14 +11,19 @@
   text-align: center;
   width: 1000px;
   margin: auto;
+  height: 100%;
 
   a {
     text-decoration: none;
   }
 }
 
 .homepage {
-  margin-top: 140px;
+  margin: auto;
+}
+
+.homepage-content {
+  margin-top: -200px;
```

Comment:
I was doing this because centering this with flexbox positioned the content a little too low. But this just made me realize that I don't even need flexbox for this so I've fixed it. Thanks for the catch! 😃